_posts/en/newsletters/2024-06-14-newsletter.md

```diff
@@ -60,39 +96,7 @@ necessary.
 
 {% assign timestamp="23:28" %}
 
-{% include functions/details-list.md
-  q0="What is the behavior change introduced by this PR?"
-  a0="Behaviour is changed in three ways. First, as per the goal of this
-  PR, optional indexes are no longer wiped again when the node is
-  restarted before reindexing is completed. This aligns the wiping
-  behavior of optional indexes with that of required indexes. Second,
-  when a user requests a reindex through the GUI, this request is no
-  longer ignored, reversing a subtle bug introduced in [b47bd95][gh
-  b47bd95]. Third, the log line \"Initializing databases...\\n\" is
-  removed."
-  a0link="https://bitcoincore.reviews/30132#l-19"
-
-  q1="What are the two ways an optional index can process new blocks?"
-  a1="When an optional index is initialized, it checks if its highest
-  block is the same as the current chaintip. If it is not, it will first
-  process all missing blocks with a background sync, through
-  `BaseIndex::StartBackgroundSync()`. When the index catches up
-  with the chaintip, it will process all further blocks through the
-  validation interface using `ValidationSignals::BlockConnected`."
-  a1link="https://bitcoincore.reviews/30132#l-52"
-
-  q2="How does this PR affect the logic of optional indexes processing
-  new blocks?"
-  a2="Before this PR, wiping the optional indexes without wiping the
-  chainstate means these indexes will be considered out-of-sync. As per
-  the previous question, that means they will first perform a background
-  sync before switching to the validation interface. With this PR, the
-  optional indexes remain in sync with the chainstate, and as such no
-  background sync is required. Note: background sync only starts after
-  reindex has completed, whereas processing validation events happens in
-  parallel."
-  a2link="https://bitcoincore.reviews/30132#l-70"
-%}
+{% include functions/pr-club.md %}
```

Comment:
bikeshed nit: this is probably personal, but I think most people refer to it as review club, so `functions/review-club.md` would be my choice